### PR TITLE
Add session/auth persistence to Chrome extension

### PR DESCRIPTION
**Pull request by twhitnah**
1. Adding a session that is created when a user logs in with google.  The session id is sent down in the response and sent back up with requests.
2. When we fetch Google auth tokens, we store then in the UserToken table.  We identify the user by email and then use that user_id for the UserToken
3. When we call plagiarism, we get the session_id and then the user and use that to get the UserToken credentials to fetch the doc

### CODE REVIEW

**Comment by twhitnah** [comment]
Current UI
Animation

Will iterate on new text flow tomorrow

**Comment by twhitnah** [comment]
Going to go ahead with merging this since there will be more iteration on top.  I think it's good to browse all PRs still though

File: api/main.py [hunk 1]
```diff
@@ -105,15 +110,15 @@ async def create():
...
         prompt_result = prompt_ref.push(
             {"articleContent": article, "createdAt": datetime.utcnow().isoformat()})
 
-        created_ref = db.reference('/createdArticles')
+        created_ref = firebase_db.reference('/createdArticles')
         created_result = created_ref.push(
             {"articleContent": full_transformed_article, "createdAt": datetime.utcnow().isoformat(), "promptId": prompt_result.key})
 
-        map_ref = db.reference('/articleMap')
+        map_ref = firebase_db.reference('/articleMap')
```
Comment (armanjaffer) [inline api/main.py:117]: Are we using firebase? My understanding was that it's an alternative to postgres

Comment (twhitnah) [reply api/main.py:117]: No, I was just renaming these variables so they didn't conflict with the db variable we're actually using.  I don't know what code you wrote that we can cull but for now I've just ignored it.

**Pull request merged**